cdci_data_analysis/analysis/parameters.py

```diff
@@ -389,6 +371,17 @@ def value(self, v):
         else:
             self._v=None
 
+    def get_value_in_default_format(self, value):
+        # might return int or float
+        if type(value) == str:
+            value = ast.literal_eval(value)
+
+        if self.default_units == 'float':
+            value = float(value)
+        elif self.default_units == 'int':
+            value = int(value)
+        return float(value)
```

Comment:
it seems as if this entire method can be reduced to just this return.

Reply:
yes, I copied from another class, and forgot to reduce it, return value is sufficient